lms/services/course.py

```diff
@@ -5,8 +5,8 @@
 
 
 class CourseService:
-    def __init__(self, ai_getter, consumer_key, db):
-        self._ai_getter = ai_getter
+    def __init__(self, application_instance, consumer_key, db):
+        self._application_instance = application_instance
```

Comment:
Do we actually want an application instance if we only access it's settings? Do we just want settings instead?

This isn't a flexibility we'd ever use in this case, but I think it's a nice idea in general if we really don't need or want other details about the parent object, as we can use this class with the sub object from any origin (not just the parent context).

Reply:
Done: https://github.com/hypothesis/lms/pull/2651/commits/bc3c8f7a6eabb42dcfbd26f7efb38974c60b4fd6

Reply:
I ended up changing this again so that `CourseService.__init__()` doesn't raise: https://github.com/hypothesis/lms/pull/2651/commits/6ad80f0e01c485252a3c2fc52d502bd7afb2156f